mmpose/visualization/opencv_backend_visualizer.py

```diff
@@ -358,7 +358,8 @@ def draw_lines(self,
                 **kwargs)
 
         elif self.backend == 'opencv':
-
+            if isinstance(colors, str):
+                colors = mmcv.color_val(colors)
```

Comment:
```suggestion
                colors = mmcv.color_val(colors)[::-1]
```

Reply:
I notice that `mmcv.color_val()` only support `bgr` mode, yet mmpose use `rgb` to visualize. So could you please also modify all `mmcv.color_val(colors)` in `opencv_backend_visualizer`?

Reply:
Ok, I see.